### PR TITLE
Production to Beta

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
     branches: [ giddh-2.0, beta-stage, beta-branch, production ]
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-concurrency: beta-build
+concurrency: prod-build
```
Comment (prateek-giddh) [inline .github/workflows/build.yml:11]: Do revert this change

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 2]
```diff
@@ -2174,6 +2174,16 @@ ul.pills li {
     }
 }
 
+.expense-ledger {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-1.scss:2177]: @aditi-tiwari17 please check this

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (ravinderwalkover) [reply .github/workflows/build.yml]: Done

File: apps/web-giddh/src/assets/css/style-1.scss (hunk 2 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/assets/css/style-1.scss:2177]: @prateek-giddh yes, this was required

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]
